res/values/strings.xml

```diff
@@ -404,6 +407,9 @@
     <string name="prefs_instant_behaviour_title">Original file will be...</string>
     <string name="upload_copy_files">Copy file</string>
     <string name="upload_move_files">Move file</string>
+    <string name="confirmation_remove_files_alert">Do you really want to remove selected items?</string>
```

Comment:
These shouldn't be added either; they were already defined in the end of the file. The build is failing because adding here they are duplicated.